arenarobot/service/processor/apriltag_detector.py

```diff
@@ -0,0 +1,175 @@
+"""
+apriltag_detector.py: reads frames, detects apriltags, and returns pose.
+
+Created by John Pedraza on 6/7/22.
+
+Copyright (c) 2022, The CONIX Research Center
+All rights reserved.
+
+This source code is licensed under the BSD-3-Clause license found in the
+LICENSE file in the root directory of this source tree.
+"""
+
+from json import JSONEncoder, dumps, loads
+import numpy as np
+import cv2
+from paho.mqtt.client import MQTTMessage
+from transformations import euler_matrix, quaternion_matrix
+from dt_apriltags import Detector
+from arenarobot.service.processor import ArenaRobotServiceProcessor
+from scipy.spatial.transform import Rotation as Ro
+import time
+
+HORIZONTAL_RESOLUTION = 3
+VERTICAL_RESOLUTION = 4
```

Comment:
Should probably be parameters with defaults.

Reply:
Perhaps I named these poorly, these are for choosing which resolution to change in OpenCV's `VideoCapture.set()` method. 3 is always for horizontal, 4 for vertical. Maybe I'll change them to `HORIZONTAL_RES_INDEX`? or something.

Reply:
Yeah `OPENCV_RES_INDEX_HORIZ` would be good.